rand_gen/src/main.rs

```diff
@@ -0,0 +1,31 @@
+extern crate rand;
```

Comment:
Please run `git rm -r rand_gen` and commit the changes so this file no longer appears